library/Exceptions/SymbolicLinkException.php

```diff
@@ -16,8 +16,11 @@
 /**
  * @author Henrique Moody <henriquemoody@gmail.com>
```

Comment:
It's just a keyword, but defining a class as final is a major change, I'd add `Gus Antoniassi <gus.antoniassi@gmail.com>` as an author here.